specs/liberty/api.rst

```diff
@@ -0,0 +1,203 @@
+      "links": {
+        "self": "http://example.gslb.openstack.org/v0.1/gslbs/f9fc6210-8efa-4492-9b4f-d87ba98f35b8",
+        "pools": "http://example.gslb.openstack.org/v0.1/gslbs/f9fc6210-8efa-4492-9b4f-d87ba98f35b8/pools",
+        "status": "http://example.gslb.openstack.org/v0.1/gslbs/f9fc6210-8efa-4492-9b4f-d87ba98f35b8/status"
+      }
+    }
+
+
+Pool JSON Snippet
+-----------------
+
+ .. code-block:: json
+
+    {
+      "id": "f9fc6210-8efa-4492-9b4f-d87ba98f35b8",
+      "name": "Main Website Pool",
+      "flavor": "example_1",
+      "status": "ACTIVE",
+      "members":["SEE MEMBER SNIPPETS BELOW"],
+      "links": {
+        "self": "http://example.gslb.openstack.org/v0.1/pools/f9fc6210-8efa-4492-9b4f-d87ba98f35b8",
+        "pool_members": "http://example.gslb.openstack.org/v0.1/pools/f9fc6210-8efa-4492-9b4f-d87ba98f35b8/pool_members",
+        "status": "http://example.gslb.openstack.org/v0.1/pools/f9fc6210-8efa-4492-9b4f-d87ba98f35b8/status"
+      }
+    }
+
+Pool Member JSON Snippet - Neutron LBaaS
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+ .. code-block:: json
+
+    {
+      "id": "f9fc6210-8efa-4492-9b4f-d87ba98f35b8",
+      "name": "Main Website",
+      "type": "neutron_lbaas_v2",
+      "region": "RegionTwo",
+      "neutron_id": "df40954c-982a-4778-a1b2-32b93cf75af9",
+      "status": "ACTIVE",
+      "endpoints": [
+        "10.10.0.1"
+      ]
+    }
+
+
+Pool Member JSON Snippet - Neutron Port
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+ .. code-block:: json
+
+    {
+      "id": "f9fc6210-8efa-4492-9b4f-d87ba98f35b8",
+      "name": "Main Website",
+      "type": "neutron_port",
+      "region": "RegionTwo",
+      "neutron_id": "78fa9c7f-f200-49df-912b-ae4679fd21e9",
+      "status": "ACTIVE",
+      "endpoints": [
+        "10.10.0.1"
+      ]
+    }
+
+Pool Member JSON Snippet - IP
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+ .. code-block:: json
+
+    {
+      "id": "f9fc6210-8efa-4492-9b4f-d87ba98f35b8",
+      "name": "Main Website",
+      "type": "IP",
+      "status": "ACTIVE",
+      "endpoints": [
+        "10.10.0.1"
+      ]
+    }
+
+
+Endpoints
+=========
+
+All of these endpoints will have the usual CRUD methods available
+
+/v0.1/gslbs
+-----------
+
+Returns a list of GLSBs
+
+/v0.1/gslbs/<uuid>
+------------------
+
+Returns a GLSB
+
+/v0.1/gslbs/<uuid>/pools
+------------------------
+
+Returns GLSB's list of pools
+
+/v0.1/gslbs/<uuid>/history
+--------------------------
+
+Returns GLSB's history (up/down/degraded etc)
+
+/v0.1/pools
+-----------
+
+Returns a list of Pools
+
+/v0.1/pools/<uuid>
+------------------
+
+Returns a pool
+
+/v0.1/pools/<uuid>/monitors
+---------------------------
+
+Returns a pool's list of monitors
+
+/v0.1/pools/<uuid>/pool_members
+-------------------------------
+
+Returns a pool's list of members
+
+/v0.1/pools/<uuid>/pool_members/<uuid>
+--------------------------------------
+
+Returns a pool member
+
+/v0.1/pools/<uuid>/pool_members/<uuid>/status
```

Comment:
Do we really need a separate "status" endpoint?  Shouldn't the pool endpoint cover it?